ts/client/scripts/risk.ts

```diff
@@ -132,13 +144,14 @@ async function computePriceImpactForLiqor(
       // Max liab of a particular token that would be liquidated to bring health above 0
       mangoAccountsWithHealth.reduce((sum, a) => {
         // How much would health increase for every unit liab moved to liqor
-        const tokenLiabHealthContrib = ONE_I80F48()
-          .sub(bank.liquidationFee)
-          .mul(bank.price)
-          .mul(bank.maintLiabWeight.sub(usdcBank.maintAssetWeight))
-          .div(usdcBank.price);
+        // liabprice * (liabweight - (1+fee)*assetweight)
+        const tokenLiabHealthContrib = bank.price.mul(
+          bank.initLiabWeight.sub(
+            ONE_I80F48().add(bank.liquidationFee).mul(usdcBank.initAssetWeight),
```

Comment:
Minor: What if the liab itself is USDC? (a lot of our borrows are) Then the asset weight of the related token would be worse than the USDC one.

But using assetweight 1 is an ok upper bound.